Makefile

```
@@ -796,6 +796,18 @@ run_experimental_migrations: bin/milmove db_deployed_migrations_reset ## Run Exp
 # ----- END PROD_MIGRATION TARGETS -----
 #
 
+#
+# ----- START PRIME TARGETS -----
+#
+
+.PHONY: run_prime_docker
+run_prime_docker:
```

We need for this to work with `make help` so it needs help text. Here's an example:

```suggestion
run_prime_docker: ## Add help text here describing what this does
```